src/packages/examples-ext/processors/exampleDependenciesBuilder.js

```diff
@@ -36,15 +39,25 @@ module.exports = function exampleDependenciesBuilder (readFilesProcessor, log, g
 
       // traverse deployments, check project dependencies and add local file
       // dependencies to the list of generated documents
-      generateExamplesProcessor.deployments.forEach(function(deployment) {
```

Comment:
I changed it from forEach to for (x in ...) because there was a case where the type of deployments was wrong. The rest of the code is as is with the exception of injecting the iframe resizer.